commands/cli/parse_test.go

```diff
@@ -259,53 +247,17 @@ func TestArgumentParsing(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-		defer os.Remove(fstdin.Name())
 
 		if _, err := io.WriteString(fstdin, content); err != nil {
 			t.Fatal(err)
 		}
 		return fstdin
 	}
-
-	test([]string{"stdinenabled", "value1", "value2"}, nil, []string{"value1", "value2"})
```

Comment:
We definitely want to have solid coverage over this. I'm not in favor of mass removing tests like this